services/vision/vision.go

```diff
@@ -48,6 +49,14 @@ type Service interface {
 	) (classification.Classifications, error)
 	// segmenter methods
 	GetObjectPointClouds(ctx context.Context, cameraName string, extra map[string]interface{}) ([]*viz.Object, error)
+	CaptureAllFromCamera(ctx context.Context,
+		cameraName string,
+		returnImage bool,
+		returnClass bool,
+		returnDet bool,
+		returnObjPCD bool,
```

Comment:
gather all of these bools into a struct, and pass the struct in as the argument. You can define this input within the viscapture package too